backend/app/fw/must/time.go

```diff
@@ -0,0 +1,16 @@
+package must
+
+import (
+	"testing"
+	"time"
+)
+
+// Time parses time from it's string representation. This simplifies test case
+// inputs involving configurable time and normalizes it to UTC.
+func Time(t *testing.T, value string) time.Time {
```

Comment:
We should refactor or delete the function mustParseTime(https://github.com/short-d/short/blob/master/backend/app/adapter/sqldb/main_test.go#L54). Opening an issue for refactoring is sufficient for now